### ISSUE TITLE
About the training

### ISSUE THREAD

**Issue by Pefect96** [levymsn/ChatIR#2]
Great job! Is the training code released?

**Comment by levymsn** [comment]
Unfortunately, not at the moment.
The training procedure is fully elaborated in the paper: We trained pairs of (dialog_i, image) in a contrastive loss manner (like CLIP), where i is the dialog length in [0, ..., 10] (excluding the initial caption). As mentioned in the paper, we used the Visual Dialog dataset (https://visualdialog.org/) for training.
Please let me know if there are any questions regarding this issue.
